packages/bpk-component-rating/README.md

```diff
@@ -12,40 +12,47 @@ npm install bpk-component-rating --save-dev
 
 ```js
 import React from 'react';
-import BpkRating, { RATING_SIZES, RATING_SCALES, RATING_TYPES } from 'bpk-component-rating';
+import BpkRating, { RATING_SIZES, RATING_SCALES } from 'bpk-component-rating';
 
 export default () => (
   <BpkRating
-    ariaLabel="9 Excellent would recommend"
+    ariaLabel="4.6 Excellent 2,420 reviews"
+    value={4.6}
     title="Excellent"
-    subtitle="It was almost perfect!"
-    value={9}
+    subtitle="2,420 reviews"
   />
 
-  // Large rating
+  // Large size rating
   <BpkRating
-    ariaLabel="9 Excellent would recommend"
+    ariaLabel="4.5 Excellent would recommend"
     title="Excellent"
     subtitle="It was almost perfect!"
     size={RATING_SIZES.large}
-    value={9}
+    value={4.5}
+  />
+
+  // Title only rating
+  <BpkRating
+    ariaLabel="4.5 Excellent would recommend"
+    title="Excellent"
+    value={4.5}
   />
 
-  // Pill-shaped rating
+  // Show max value rating
   <BpkRating
-    ariaLabel="9 Excellent would recommend"
+    ariaLabel="4.8 Excellent would recommend"
     title="Excellent"
     subtitle="It was almost perfect!"
-    type={RATING_TYPE.pill}
-    value={9}
+    value={4.8}
+    showMaxValue
   />
 
-  // 0-5 scale
+  // 0-10 scale
   <BpkRating
-    ariaLabel="2 It was ok I guess"
```

Comment:
I noticed that we have no other **low rating** samples in other tests or samples except this deleted one. I suggest we can consider adding it in the test file or readme.

Reply:
Actually, we have low score rating test case in [BpkRating-test L88]( https://github.com/Skyscanner/backpack/pull/2430/files#diff-1be9937911203ef3c904db1c089f154871bbfd3f7f7bdb0c13b1fe534cfcd3f0L75 )